src/mrpro/operators/_Functional.py

```diff
@@ -0,0 +1,48 @@
+"""Base Class Functional."""
+
+from abc import ABC
+from abc import abstractmethod
+from collections.abc import Sequence
+
+import torch
+
+from mrpro.operators._Operator import Operator
+
+
+class Functional(Operator[torch.Tensor, tuple[torch.Tensor]]):
+    """Functional Base Class.
+
+    Args:
+        Operator (_type_): _description_
+    """
+
+    def __init__(
+        self, lam: torch.Tensor | float = 1.0, g: torch.Tensor | None = None, dim: Sequence[int] | None = None
```

Comment:
Please use more descriptive names for lam and g

Reply:
I would also suggest to remove the parameter "lambda" in the the init, as it is only required when computing the prox, since we are usually interested in computing the prox of lambda*|| ||_1.

Reply:
Does it makes sense to have the data g as an argument in the init of the class?

Reply:
without the lambda, we have no way to specify lambda*|| ||_1.
it should also be used in the forward.

Reply:
but is this needed in the first place?

Reply:
Yes, if we want to ever allow lambda as maps.